Example/EpoxyExample/Views/LayoutGroups/Elements/ImageView.swift

```diff
@@ -0,0 +1,54 @@
+// Created by Tyler Hedrick on 1/27/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCore
+import UIKit
+
+final class ImageView: UIImageView, EpoxyableView {
```

Comment:
nit: seems like we're using this more for an icon since it has a tint color
```suggestion
final class IconView: UIImageView, EpoxyableView {
```